deployment/cdk/opensearch-service-migration/test/migration-console-stack.test.ts

```diff
@@ -81,7 +81,7 @@ test('Test that services yaml parameter is created', () => {
     const yamlFileContents = value['Fn::Join'][1].join('')
     expect(yamlFileContents).toContain('source_cluster')
     expect(yamlFileContents).toContain('target_cluster')
-    expect(yamlFileContents).toContain('metrics_source:\n  type: cloudwatch')
```

Comment:
Is cloudwatch null here because we haven't set it up in our services.yaml with CDK yet?

Reply:
No, it's totally set up and working (this is basically what the cloud deployment option tests).

It's actually because all we care about is the presence of the key. So the YAML _should_ look like:
```
metrics:
  cloudwatch:
```
or, if an optional value is provided,
```
metrics:
  cloudwatch:
    region: us-east-1
```
but an empty value = null, and the YAML library dumps it with the explicit type. But I agree it's confusing. 

This comment prompted me to go down a little mini rabbit hole, and I think it's actually possible to change the dump preferences, so i'm going to do a quick check with that and see if that makes it much clearer.